quickstart/201-vmss-disk-encryption-extension/main.tf

```diff
@@ -106,7 +110,7 @@ resource "azurerm_windows_virtual_machine_scale_set" "example" {
 }
 
 // Disk Encryption Extension
-resource "azurerm_virtual_machine_scale_set_extension" "example" {
+resource "azurerm_virtual_machine_scale_set_extension" "main" {
```

Comment:
I think this example's leading resource is still `azurerm_windows_virtual_machine_scale_set`, so would you please rename this extension resource to `example`, and rename `azurerm_windows_virtual_machine_scale_set.example` to `azurerm_windows_virtual_machine_scale_set.main`? Thanks